### PR TITLE
PKG-1321 update to 2023.3.2

### PR DESCRIPTION
**Pull request by M-Waszkiewicz-Anaconda**

### CODE REVIEW

**Review by skupr-anaconda — APPROVED** [review]
LGTM.
Note: dask-core --->distributed --->dask should be uploaded at the same day

**Comment by M-Waszkiewicz-Anaconda** [comment]
> 

yes, working on it now

**Comment by akabanovs** [comment]
Also, recipe is missing description

**Comment by M-Waszkiewicz-Anaconda** [comment]
> Also, recipe is missing description

done

**Comment by M-Waszkiewicz-Anaconda** [comment]
> Can remove "doc_source_url" and use https://docs.dask.org in doc_url.

done

File: recipe/meta.yaml [hunk 1]
```diff
@@ -25,12 +23,14 @@ requirements:
     - wheel
   run:
     - python >=3.8
```
[the remark this answers is not in the document]
Comment (M-Waszkiewicz-Anaconda) [reply recipe/meta.yaml]: > No need to specify the pinning for python, I think.

done

**Pull request merged**